backend/models/courseunit.mgmodel.ts

```diff
@@ -0,0 +1,105 @@
+import mongoose, { Schema, Document } from "mongoose";
+
+type ElementSkeleton = {
```

Comment:
Could we `export` the type definitions for `ElementSkeleton`, `Page`, and `Module`? I'm thinking our backend operations will likely also make use of these types once we start implementing our APIs